internal/server/testdata/openapi3.json

```diff
@@ -0,0 +1,6957 @@
+{
```

Comment:
This file was removed. Bad rebase with the changes in #3551 ?

Reply:
Removed 👍